src/sfx/sfx.hpp

```diff
@@ -2,17 +2,14 @@
 
 #include "en/Node.hpp"
 
-namespace ari::en
+namespace ari::sfx
 {
-	namespace sfx
+	class Sfx : public Node
```

Comment:
What is this component for?

Reply:
Other component such as AudioSource and AudioListener would be derived from Sfx and can be behave as a node.